src/services/notifications/notifications.test.ts

```diff
@@ -0,0 +1,190 @@
+import app from '../../app';
+import { expect } from 'chai';
+
+const service = app.service('notifications');
+const userInfo = {
+  firstName: 'test user name',
+  lastName: 'test user last',
+  email: 'create_notification@example.com',
+  password: 'supersecret',
+};
+let userId!: number;
+let notificationId: number;
+const notExistedNotificationId = '66c9a620-3bdc-11ec-a993-ff3e9e1971e7';
+const notificationInfo = {
+  description: 'test description',
+  userId,
+  type: 'REPORT_CREATED',
+};
+
+describe('\'notifications\' service', () => {
+  it('registered the service', () => {
+    expect(service).to.exist;
+  });
+  describe('create notification', () => {
+    before(async () => {
+      const { id } = await app.service('users').create(userInfo);
+      userId = id;
+      notificationInfo.userId = id;
+    });
+
+    after(async () => {
+      await service.remove(notificationId);
+      await app.service('users').remove(userId);
+    });
+
+    it('should succeed - create notification', async () => {
+      const notificationResponse = await service.create(notificationInfo);
+      const { description, id } = notificationResponse;
+      notificationId = id;
+      expect(id).to.exist;
+      expect(description).to.equal(notificationInfo.description);
+      expect(notificationResponse).to.have.keys(['id', 'isRead', 'type', 'description', 'createdAt', 'updatedAt', 'userId']);
+    });
+
+    it('should fail - not existed userId', async () => {
+      const notificationInvalidUserId = {
+        description: 'test notification',
+        userId: '783deee0-3732-11ec-9301-13adbda06b66',
+      };
+      try {
+        await service.create(notificationInvalidUserId);
+        expect.fail('call should have failed');
+      } catch (err: any) {
+        const { code } = err;
+        expect(code).to.equal(400);
+      }
+    });
+
+    it('should fail - invalid data', async () => {
+      const notificationInvalidData = {
+        description: 12,
+        isRead: 'yes',
+        type: 'READ_WRITE',
+      };
+      try {
+        await service.create(notificationInvalidData);
+        expect.fail('call should have failed');
+      } catch (err: any) {
+        const { code } = err;
+        expect(code).to.equal(400);
```

Comment:
check error messages if possible

Reply:
add check for error message 9978e7fc0bb98929f17ca5c20384f8b3d56f0792